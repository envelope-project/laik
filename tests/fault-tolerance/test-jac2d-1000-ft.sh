#!/bin/sh

EXECUTABLE="../../examples/fault-tolerance/checkpoint-jac2d-recovery"
mpirun -v --oversubscribe -n 8 "$EXECUTABLE" --failureCheckFrequency 10 --checkpointFrequency 10 --redundancyCount 1 --rotationDistance 1 --plannedFailure 1 15 --plannedFailure 2 21  --plannedFailure 3 32 --plannedFailure 5 33 -s 1000 > test-jac2d-1000-ft.out
cmp test-jac2d-1000-ft.out "$(dirname -- "${0}")/test-jac2d-1000-ft.expected"