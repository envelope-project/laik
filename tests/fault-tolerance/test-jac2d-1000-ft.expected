1000 x 1000 cells (mem 16.0 MB), running 50 iterations with 8 tasks
Residuum after  1 iters: 3007147.625000
Residuum after 11 iters: 2377.016846
Residuum after 11 iters: 2377.016846
Residuum after 21 iters: 150.808462
Residuum after 21 iters: 150.808462
Residuum after 31 iters: 82.356528
Residuum after 31 iters: 82.356528
Residuum after 41 iters: 53.986431
Global value sum after 50 iterations: 2946003.362703
